util/coverage/coverage.js

```diff
@@ -210,6 +210,7 @@ const miscStrings = {
   // Description about release and latest version differences.
   description: {
     en: 'This list may contain content that is in development and is not yet included in the latest cactbot release. Anything that is listed as covered here will be included in the next release of cactbot.  If you are using the <a href="https://github.com/quisquous/cactbot/blob/main/CONTRIBUTING.md#validating-changes-via-remote-urls">quisquous.github.io version</a> as the url for your overlays, this list will be up to date.',
+    de: 'Diese Liste kann Inhalte beinhalten, welche momentan in entwicklung sind uns sich noch nicht im aktuellsten Cactbot Release befinden. Alles was hier aufgelistet ist, wird sich im nächsten Release von Cactbot befinden. Wenn du <a href="https://github.com/quisquous/cactbot/blob/main/CONTRIBUTING.md#validating-changes-via-remote-urls">quisquous.github.io version</a> als URL für dein Overlay benutzt, ist diese Liste bereits aktuell für dich.',
```

Comment:
entwicklung -> Entwicklung
uns -> und

Reply:
Maybe also consider changing "Inhalte" to "Punkte" to avoid having "Inhalte beinhalten" directly after another?

Reply:
aktuellsten -> aktuellstem

ist diese Liste bereits aktuell für dich -> sind die Punkte auf dieser Liste für Dich bereits verfügbar.

Reply:
i would leave Inhalte, as Punkte does not reflect the reason of the list.
Changed beinhalten to enthalten so its a little bit less similar